### DISCUSSION TITLE
How to cite PPOCRLabel

### DISCUSSION THREAD

**Discussion by zengyue1376** [PaddlePaddle/PaddleOCR#12322] [Q&A]
How to cite PPOCRLabel in paper? Is there any paper published?

**Comment by YoucanBaby** [comment]
same question

**Comment by SWHL** [comment]
Reference labelImg project([link](https://github.com/HumanSignal/labelImg#:~:text=Citation%3A%20Tzutalin.%20LabelImg.%20Git%20code%20(2015).%20https%3A//github.com/tzutalin/labelImg)), maybe:

Citation: PaddleOCR. PPOCRLabel. Git code (2020). https://github.com/PFCCLab/PPOCRLabel
